lib/endpoints/class-wp-rest-attachments-controller.php

```diff
@@ -122,10 +123,11 @@ public function update_item( $request ) {
 			update_post_meta( $data['id'], '_wp_attachment_image_alt', $request['alt_text'] );
 		}
 
-		$response = $this->get_item( array(
-			'id'      => $data['id'],
-			'context' => 'edit',
-		));
+		$attachment_location = rest_url( '/wp/v2/' . $this->get_post_type_base( get_post_type( $data['id'] ) ) . '/' . $data['id'] );
```

Comment:
Is it necessary that we pass the method and url to `WP_REST_Request`? Given we are calling `get_item` I don't think it should be required to have a "correct" `WP_REST_Request` for that function. `get_item` just says "get me one of these items, this request object is the source request, by the way". Given we didn't need to generate the object URLs before, I don't see why we need to do it now.